types/hapi__joi/index.d.ts

```diff
@@ -190,6 +194,17 @@ declare namespace Joi {
         warnings?: boolean;
     }
 
+    interface LanguageMessageTemplate {
+        source: string;
+        rendered: string;
+        _template: null | any[];
```

Comment:
`_` usually means internal and should not be exposed in typings.

Reply:
_template and _settings have been removed from interface.